share/cmake/MrDocs.cmake

```diff
@@ -146,6 +146,12 @@ function(add_mrdocs MRDOCS_TARGET_NAME)
         set(MRDOCS_TARGET_OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/mrdocs)
     endif()
 
+    #-------------------------------------------------
+    # LibC++ paths
+    #-------------------------------------------------
+    if (NOT DEFINED MRDOCS_LIBCXX_PATHS)
+        set(MRDOCS_LIBCXX_PATHS ${CMAKE_SOURCE_DIR}/share/libcxx)
```

Comment:
I think 1) this script is not being supported and 2) if it were, the logic above is wrong.

I'm not even sure we should be maintaining this script. Klemens wrote this once, but it's kind of a workaround, and we haven't been supporting (testing) it. It's a huge workaround because it only makes sense when the target is there. In other words, it only makes sense inside the MrDocs project. 

In practice, if people want to create a target that generates MrDocs documentation, I expect them to have MrDocs installed and create a custom target with whatever command they want. Then we could have a MrDocs.cmake script bundled with the package to help them create these custom targets (like Catch2 bundles a script to create test targets) and let people use it. 

But this would need to be tested. For now, I think we can think of this script as deprecated because we're not really supporting or testing it.

In any case, if some other project includes this script, it assumes the other project has a copy of libcxx in `${CMAKE_SOURCE_DIR}/share/libcxx`, which is not the case. Even MrDocs shouldn't have a copy of libcxx there.